hs_access_control/models.py

```diff
@@ -231,12 +342,16 @@ def create_group(self, title):
         owner.
         """
         if __debug__: 
-            assert isinstance(title, basestring) 
+            assert isinstance(title, basestring)
+            assert isinstance(description, basestring)
+            if purpose:
+                assert isinstance(purpose, basestring)
```

Comment:
@pkdash Do we really need these assert statements as part of the code base? It makes sense to do type checking when debugging, but I feel it is better to add the types into docstring of the method rather than having these assert statements as part of the code.